pkgs/by-name/ma/mas/package.nix

```diff
@@ -36,13 +35,7 @@ stdenvNoCC.mkDerivation rec {
     description = "Mac App Store command line interface";
     homepage = "https://github.com/mas-cli/mas";
     license = licenses.mit;
-    maintainers = with maintainers; [
-      steinybot
-      zachcoyle
-    ];
-    platforms = [
-      "x86_64-darwin"
-      "aarch64-darwin"
-    ];
+    maintainers = with maintainers; [ steinybot zachcoyle ];
+    platforms = [ "x86_64-darwin" "aarch64-darwin" ];
```

Comment:
`lib.platforms.darwin`?

Reply:
I considered this, but it was changed _from_ `lib.platforms.darwin` in https://github.com/NixOS/nixpkgs/commit/e55eb3d376c05d2bc902506aa0c42c8eb8c06bcb#diff-e9cddb1ec8d2c67121f7529df3f0326832437dea4108f3640057e02053ad5227L36. I assumed it was because the other `-darwin` systems aren’t supported.